### ISSUE TITLE
Posters not appearing in the Shift view in Estuary if HDD not connected

### ISSUE THREAD

**Issue by BungeeG** [xbmc/xbmc#21744]
### Describe the bug
Here is a clear and concise description of what the problem is:

Posters don't appear in the Shift view in the Estuary skin if the HDD which contains the local information is disconnected

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

The posters should appear even if the HDD is disconnected (as in other views)

## Actual Behavior

The posters don't appear

## Possible Fix

I know how to fix, you have to edit the View_53_Shift.xml file and edit the 72 and 106 textures lines by adding this :
background="true"

### To Reproduce

1. Scrape information with local information only from a HDD
2. Exit Kodi and disconnect the HDD
3. Launch Kodi with the HDD disconnected
4. Go to the Shift view and see that the posters are not appearing

### Debuglog

None

### Screenshots 
Here are some links or screenshots to help explain the problem:

None

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

None

### Your Environment
Used Operating system:

 - [x] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] OSX
 - [x] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version: Matrix 19.4

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by sarbes** [comment]
Please adhere to the template. No log, no problem - we don't want to guess what's going on on your system.

**Comment by BungeeG** [comment]
It's not really an issue, just something kodi developpers forget to add in a line in the View_53_Shift.xml file of the estuary skin. So you won't find anything in the log file. It's on every system. The problem is that, the artworks will have to load every time the artwork is displayed so it will have to find and use the HDD every time. But if you add background="true" to the texture line then it will be always loaded and won't have to find the artwork in the HDD everytime.

**Comment by howie-f** [comment]
>But if you add background="true" to the texture line then it will be always loaded

i've tested this and can confirm. @BungeeG would you like to fix this with your first pullrequest?

**Comment by BungeeG** [comment]
Thanks for your answer, I tried to figure out how to do a pullrequest but I think it's not as easy as opening a new issue post. Can you create the pull request ? Or at least tell me exactly how to do it ?

**Comment by howie-f** [comment]
ok, opened here https://github.com/xbmc/xbmc/pull/21749 bur i missed your

> Also there is the exact same problem in the DialogVideoInfo.xml file

comment :) no need to open a new issue, i think we can include this in the pr. any idea if there's more places where the `background="true"` is missing?

**Comment by sarbes** [comment]
> It's not really an issue, just something kodi developpers forgot to add in a line in the View_53_Shift.xml file of the estuary skin. So you won't find anything in the log file. It's on every system. 

Assumptions. 

From my understanding, there should not be any functional differences in both texture loaders. So yes, it is a bug where the debug log is relevant.

**Comment by BungeeG** [comment]
Yes, as I said in the DialogVideoInfo.xml file, the poster is not always loaded. And in the Custom_1104_Fanart.xml file, the texture line also need a background="true". Also in the dialog video info, if you click on the "Choose art" button, the artworks won't be loaded so you have to edit the two $VAR[InfoWallThumbVar] lines in the Includes_DialogSelect.xml file to add background="true".



> > It's not really an issue, just something kodi developpers forgot to add in a line in the View_53_Shift.xml file of the estuary skin. So you won't find anything in the log file. It's on every system.
> 
> Assumptions.
> 
> From my understanding, there should not be any functional differences between both texture loaders. So yes, it is a bug where the debug log is relevant.

Is it possible developpers didn't add background="true" on purpose ?

**Comment by howie-f** [comment]
hum, if i get it right it shouldn‘t matter whether `background=true` is set or not. so @BungeeG it would be nice if you could generate a debuglog and attach it here. you can use https://paste.kodi.tv/

**Comment by BungeeG** [comment]
Here is it : https://paste.kodi.tv/sekiliwili.kodi
